nengo_gui/guibackend.py

```diff
@@ -102,17 +103,47 @@ class GuiRequestHandler(server.HttpWsRequestHandler):
         '/favicon.ico': 'serve_favicon',
     }
 
-    def get_expected_origins(self):
-        session = self.get_session()
+    def _is_loopback(self, host):
```

Comment:
Could you add a quick docstring about what this function is doing? From what I can parse this is used to verify if the expected host was reached? This function only seems to get called in `assert` statements.